{{cookiecutter.hyphenated}}/catalog-info.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: {{ cookiecutter.hyphenated }}
+  title: {{ cookiecutter.project_name }}
+
+  description: {{ cookiecutter.project_short_description}}
```

Comment:
Nit - the spacing of `}}` on this line looks different from the rest. Probably doesn't impact anything, but it caught my eye.

Reply:
That would bother me as well, I'll correct it.